app/pages/multisig/updates/common/util.ts

```diff
@@ -19,3 +19,18 @@ export const lengthRule: (
 
 export const mustBeAnInteger: Validate = (v) =>
     onlyDigitsNoLeadingZeroes(v) || 'Must be a valid integer';
+
+export const validationRulesForPositiveWord64 = (name: string) => ({
+    required: requiredMessage(name),
+    min: {
+        value: 1,
+        message: `${name} must be positive`,
```

Comment:
We probably want to capitalize `${name}`.

Reply:
I think that should be up to the name giver :shrimp: